e2eTests/e2eTestPrompts.md

```diff
@@ -0,0 +1,10 @@
+This file contains prompts used for end-to-end testing to ensure each tool is invoked properly by LLM. This table is organized by alphabetical order of the tool names.
```

Comment:
At the moment all our commands live interspersed, so this works. @hallipr is working on separating each service into its own folder. Would it be possible to split these e2e prompts into service specific prompts?

Reply:
It is mapped per tool/command. We can definitely split this table by service into multiple files if that's the desirable behavior in the future.